src/app/api/v1/admin/feature-flags/route.ts

```diff
@@ -24,7 +24,7 @@ export async function GET() {
     try {
       const flags = await getAllFeatureFlags();
       return NextResponse.json(flags);
-    } catch (e) {
```

Comment:
I know you didn't introduce this and I'm fine with the change as-is, but I think there's value in logging these messages so we can tell exactly what went wrong when diagnosing. I don't think that it should be returned to the caller or anything.

Reply:
I'm not sure how to tackle this, we might want a tech debt ticket to audit our existing exception handling. wdyt?